core.Dockerfile

```diff
@@ -37,6 +37,19 @@ RUN mkdir -p .kcp && \
     git config --global --add safe.directory /home/kubestellar && \
     mkdir -p bin
 
+RUN git clone https://github.com/kube-bind/kube-bind.git && \
+    pushd kube-bind && \
+    IGNORE_GO_VERSION=1 make build && \
+    popd && \
+    git clone -b autobind https://github.com/waltforme/kube-bind.git kube-bind-autobind && \
+    pushd kube-bind-autobind && \
+    IGNORE_GO_VERSION=1 go build -o ../kube-bind/bin/kubectl-bind cmd/kubectl-bind/main.go && \
```

Comment:
https://github.com/waltforme/kube-bind is a full copy, right? Why not just build all of kube-bind from that?

Reply:
Yes, changed and now we are building all from that.